tests/runtime/file-output

```diff
@@ -0,0 +1,19 @@
+NAME printf to file
+RUN bpftrace -v -e 'i:ms:10 { printf("%s %d\n", "SUCCESS", 1); exit() }' -o bpftrace-file-output-test >/dev/null; cat bpftrace-file-output-test; rm bpftrace-file-output-test
```

Comment:
I also think it's important to put this file we're writing to in `/tmp` so that it gets auto cleaned up eventually if our tests fail and the user will definitely have permission to write to it

Reply:
I had changed it but failed to push the latest version. 

The runtime tests need to be run as root so I'm not sure when write permissions would be an issue. However, I dont see any issues with putting it in /tmp instead of the build dir.

Reply:
I was thinking maybe some capabilities would allow bpftrace to run without root, but I don't really know. Also we've been having some problems with snap file permissions recently (not for the tests though, of course)